packages/widgets/src/DataTable/DataTableColumnHeader.js

```diff
@@ -0,0 +1,154 @@
+import { requiredIf } from '@dhis2/prop-types'
+import cx from 'classnames'
+import PropTypes from 'prop-types'
+import React, { forwardRef } from 'react'
+import styles from './DataTableColumnHeader/DataTableColumnHeader.styles.js'
+import { FilterHandle } from './DataTableColumnHeader/FilterHandle.js'
+import { Sorter } from './DataTableColumnHeader/Sorter.js'
+import { TableHeaderCell } from './TableElements/index.js'
+
+const SORT_DIRECTIONS = ['default', 'asc', 'desc']
```

Comment:
I think this should be imported from `./DataTableColumnHeader/Sorter.js` instead of being defined again.